pulsar-common/src/main/proto/PulsarApi.proto

```diff
@@ -302,6 +302,7 @@ message FeatureFlags {
   optional bool supports_partial_producer = 3 [default = false];
   optional bool supports_topic_watchers = 4 [default = false];
   optional bool supports_get_partitioned_metadata_without_auto_creation = 5 [default = false];
+  optional bool supports_dedup_repl_v2 = 6 [default = false];
```

Comment:
There is no document for how "v2" makes the difference from the existing behavior and why it needs two versions of replicator deduplication. I think it definitely requires a PIP to explain it in details

Reply:
Renamed it to `supports_repl_dedup_by_lid_and_eid`